web_scraper/web_scraper/pipelines.py

```diff
@@ -0,0 +1,13 @@
+# Define your item pipelines here
```

Comment:
Framework Code- Automatically created

Reply:
So these files are auto-generated and we shouldn't change it right?

Reply:
Yes